packages/ckeditor5-mention/tests/mentionui.js

```diff
@@ -131,6 +131,41 @@ describe( 'MentionUI', () => {
 		} );
 	} );
 
+	describe( 'contextual balloon in rtl', () => {
+		let getBalloonPanelPositionDataSpy;
+
+		beforeEach( async () => {
+			await createClassicTestEditor( { ...staticConfig } );
+
+			setData( model, '<paragraph>foo []</paragraph>' );
+			editor.locale.uiLanguageDirection = 'rtl';
+			getBalloonPanelPositionDataSpy = sinon.spy( mentionUI, '_getBalloonPanelPositionData' );
+
+			model.change( writer => {
+				writer.insertText( '@', doc.selection.getFirstPosition() );
+			} );
+
+			await waitForDebounce();
+		} );
+
+		afterEach( () => {
+			editor.locale.uiLanguageDirection = 'ltr';
+		} );
```

Comment:
I suppose this could be removed. Every test group creates its own editors and the override should not leak.